extensions/target-specific/chromium/blink/fragment-tree/fragment-tree.js

```diff
@@ -27,8 +27,15 @@ Loader.OnLoad(function() {
             }
         },
         GetRoots: function() {
-            return BlinkHelpers.GetDocuments()
-            .then((documents) => {
+            return Promise.all([BlinkHelpers.GetDocuments(), DbgObject.global(Chromium.RendererProcessSyntheticModuleName, "is_layout_ng_enabled_", "bool", "blink::RuntimeEnabledFeatures").val()])

```

Comment:
I think it would be better to check if LayoutNG is enabled before calling GetDocuments, instead of using Promise.all. That way, if LayoutNG is disabled, we won't pay the unnecessary cost of querying g_frame_map.

Reply:
Done